api/common/upgradeseries.go

```diff
@@ -48,13 +51,23 @@ func (u *UpgradeSeriesAPI) WatchUpgradeSeriesNotifications() (watcher.NotifyWatc
 	return w, nil
 }
 
-// UpgradeSeriesStatus returns the upgrade series status of a unit from remote state
-func (u *UpgradeSeriesAPI) UpgradeSeriesStatus() (string, error) {
+// UpgradeSeriesPrepareStatus returns the upgrade series status of a unit from remote state
```

Comment:
Wrong method name.

I know the prepare/complete status is set to converge, it seems janky to have 2 methods at he API server that hide passing the status type, and then to expose the type client side and call the server methods that hide it.